scenes/quests/lore_quests/quest_003/1_stealth_sequence_combo/stealth_sequence_combo.tscn

```diff
@@ -0,0 +1,290 @@
+[ext_resource type="AudioStream" uid="uid://bwif2oo6ymiu2" path="res://assets/third_party/sounds/characters/enemies/guard/TorchHit.ogg" id="10_f0rs2"]
+[ext_resource type="SpriteFrames" uid="uid://d1k25io0tiiij" path="res://scenes/game_elements/characters/enemies/guard/components/storyvoretorch_frames_blue.tres" id="11_bc7ib"]
+[ext_resource type="Script" uid="uid://c68oh8dtr21ti" path="res://scenes/game_logic/sequence_puzzle.gd" id="14_qhjva"]
+[ext_resource type="Resource" uid="uid://c6npgkuhqql7" path="res://scenes/quests/lore_quests/quest_003/1_stealth_sequence_combo/components/collected.dialogue" id="15_00lb2"]
+[ext_resource type="Script" uid="uid://ccc78coj2b1li" path="res://scenes/game_logic/sequence_puzzle_step.gd" id="15_aaq2u"]
+[ext_resource type="PackedScene" uid="uid://b8sok264erfoc" path="res://scenes/game_elements/props/sequence_puzzle_object/sequence_puzzle_object.tscn" id="15_lxyj7"]
+[ext_resource type="SpriteFrames" uid="uid://7ul4p7v1ve0p" path="res://scenes/quests/lore_quests/quest_001/1_music_puzzle/components/musical_rock_sprite_frames.tres" id="16_4t55h"]
+[ext_resource type="AudioStream" uid="uid://byi3b8aywp6py" path="res://assets/third_party/nepalese_hand_bells/handBells-a3.ogg" id="17_qhjva"]
+[ext_resource type="AudioStream" uid="uid://cmtiwg2cylmts" path="res://assets/third_party/nepalese_hand_bells/handBells-e4.ogg" id="18_0cbql"]
+[ext_resource type="SpriteFrames" uid="uid://blhuke03t517u" path="res://scenes/quests/lore_quests/quest_001/1_music_puzzle/components/bonfire_sign/bonfire_sign_blank.tres" id="20_5dmi6"]
+[ext_resource type="AudioStream" uid="uid://cg57q82pb243w" path="res://assets/third_party/nepalese_hand_bells/handBells-c4.ogg" id="20_46vvk"]
+[ext_resource type="AudioStream" uid="uid://shckj7dsimyt" path="res://assets/third_party/nepalese_hand_bells/handBells-g#4.ogg" id="20_lxyj7"]
+[ext_resource type="PackedScene" uid="uid://be4o3ythda4cu" path="res://scenes/game_elements/props/sequence_puzzle_hint_sign/sequence_puzzle_hint_sign.tscn" id="22_4t55h"]
+[ext_resource type="PackedScene" uid="uid://dv4f232y8w8dv" path="res://scenes/game_elements/props/decoration/water_rock/water_rock.tscn" id="25_bc7ib"]
+
+[sub_resource type="Curve2D" id="Curve2D_vnsq3"]
+_data = {
+"points": PackedVector2Array(0, 0, 0, 0, -3500, 2692.99, 0, 0, 0, 0, 2034.66, 2724.17, 0, 0, 0, 0, 2128, 3523.56, 0, 0, 0, 0, 2127.99, 2729.84, 0, 0, 0, 0, 4610.66, 2729.84)
+}
+point_count = 5
+
+[sub_resource type="Curve2D" id="Curve2D_qhjva"]
+_data = {
+"points": PackedVector2Array(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1111, -13, 0, 0, 0, 0, 1109, -380, 0, 0, 0, 0, 1416, -392)
+}
+point_count = 4
+
+[sub_resource type="Resource" id="Resource_idy4y"]
+script = ExtResource("9_1l5nh")
+metadata/_custom_type_script = "uid://bgmwplmj3bfls"
+
+[node name="StealthSequenceCombo" type="Node2D"]
+y_sort_enabled = true
+metadata/patchwork_id = "e0d6d619601849f5be54868c6b970eb3"
```

Comment:
These IDs come from the Patchwork plugin, where we initially prototyped this level.

I am deliberately leaving them here because I want to see if there are any ill-effects of doing so.